design-toolkit/components/src/components/button/index.tsx

```diff
@@ -64,7 +79,7 @@ export function Button({ ref, ...props }: ButtonProps) {
     className,
     color = 'mono-muted',
     size = 'medium',
-    variant,
+    variant = 'flat',
```

Comment:
is it correct to do this? the kitchen sink buttons that didn't specify a variant looked funky